### PR TITLE
Remove custom policies / clean up styles based on feedback from design

### LINKED ISSUE #88: Other clean up items after talking with design
- Remove timestamp
- hide download button (check first)
- pie chart is responsive
- compliance pie charts need to be same size as patch
- compliance pie charts change colors to match patch
- compliance - wrap the label data
- change font-size of bottom link in advisor

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #88 #85 #84

## Prerequisites

- [X] Scope your styles to your individual card
- [X] You are using translations when necessary with proper plural/singular modifications
- [X] You are using the template card provided
- [X] Use functional components & hooks (please)
- [X] Attach screenshots (before and after)
- [X] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What

- Remove timestamp
- pie chart text shouldn't be changing
- compliance pie charts need to be same size as patch
- compliance pie charts change colors to match patch
- change font-size of bottom link in advisor
- moved styles out of `dashboard.scss` and into the relative card folders

### Before
Screen Shot 2020-03-31 at 2 29 37 PM
Screen Shot 2020-03-31 at 2 29 41 PM

### After
Screen Shot 2020-03-31 at 2 24 24 PM

Screen Shot 2020-03-31 at 2 24 35 PM

## Additional Info
[if needed]

## Code Review
@christiemolloy @ryelo @AllenBW

## Design Review
@kybaker

### CODE REVIEW

File: src/ChartTemplates/PieChart/PieChartTemplate.scss [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
 .ins-c-pie-chart__row {
     display: flex;
     flex-direction: row;
 }
 
-.ins-c-pie-chart__legend {
-    padding-top: 12px;
-    padding-left: 15px;
-}
+// .ins-c-pie-chart__legend {
```
Comment (AllenBW) [inline src/ChartTemplates/PieChart/PieChartTemplate.scss:6]: an we kill the commented code (and file?)

File: src/PresentationalComponents/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -70,11 +67,6 @@ const Dashboard = () => {
                         { permission.remediations ? <RemediationsCard /> : <DeniedState appName='Remediations'/> }
                     </Suspense>
                 </div>
-                <div className="dashboard-card-custom-policies">
```
Comment (AllenBW) [inline src/PresentationalComponents/Dashboard/Dashboard.js:73]: farewell custom policies card, may you rest in peace 💀

File: src/SmartComponents/Compliance/ComplianceCard.js [hunk 3]
```diff
@@ -35,6 +36,7 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
 
     const intl = useIntl();
     const pieChartPadding = { bottom: 0, left: 0, right: 0, top: 0 };
+    const colorScale = ['#06c', '#519de9'];
```
Comment (AllenBW) [inline src/SmartComponents/Compliance/ComplianceCard.js:39]: can we use`@patternfly/react-tokens` here

File: src/SmartComponents/PatchManager/PatchManagerCard.js [hunk 4]
```diff
@@ -29,14 +29,14 @@ const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity,
 
     const pieChartData = [
         { x: intl.formatMessage(messages.securityAdvisories, { count: security }), y: security, fill: '#004b95' },
-        { x: intl.formatMessage(messages.bugfixAdvisories, { count: bugs }), y: bugs, fill: '#06c' },
-        { x: intl.formatMessage(messages.enhancementAdvisories, { count: enhancements }), y: enhancements, fill: '#519de9' }
+        { x: intl.formatMessage(messages.bugfixAdvisories, { count: bugs }), y: bugs, fill: '#519de9' },
```
Comment (AllenBW) [inline src/SmartComponents/PatchManager/PatchManagerCard.js:32]: ditto above,  use@patternfly/react-tokens here

**Comment by kybaker** [comment]
@AllenBW @christiemolloy Can we reduce the height of the boxes in the top row. I suspect this was an accidental change. Otherwise looks great

**Comment by christiemolloy** [comment]
hm I wasn't seeing any increase in height in the top row on my screen, thanks for pointing that out @AllenBW , I might need to add a max-height @kybaker

**Comment by christiemolloy** [comment]
updated @AllenBW  , since I dont have my monitor im not able to stretch my screen to as tall as you have in the screenshot above, however I added a max-height to the first card group so that it wont grow any taller than what it is now (200px). Row 2 and 3 should continue to stretch as you have in the screenshot (it matches Kyles mocks). cc: @kybaker

**Comment by AllenBW** [comment]
@christiemolloy if yah get to that other feedbacks, the number things, this'll look great!

**Pull request merged**